jobs/nessus-manager/templates/bin/emit-scans.sh

```diff
@@ -49,3 +49,12 @@ for scanid in $(curl -sk https://127.0.0.1:8834/scans \
       -H "X-Cookie: token=${token}"
   done
 done
+
+# if we reached here, then we didn't bomb out in some way because the API
+# changed or something, so let's log that success so that we can alert if
+# this gets old.
+cat <<EOF > /var/vcap/jobs/node_exporter/config/nessusscandelete.prom
```

Comment:
I've been basing other textfile examples on a blog post from one of the prometheus developers: https://www.robustperception.io/monitoring-directory-sizes-with-the-textfile-collector/. They're careful to write metrics to a temp file, then move to the node exporter directory after writing, because the move is atomic, but writing the file might not be--the exporter might try to grab metrics in the middle of a write otherwise. That's what we do in https://github.com/18F/cg-nessus-manager-boshrelease/blob/master/jobs/nessus-manager/templates/bin/health.sh#L9-L19. Does that make sense?

Reply:
It does, though I think it is silly to worry about that, because it's _such_ an edge case.  Even if you managed to get a filedescriptor open in between the open() and the close(), you'd only get an empty file.  There's no way we'd ever get a truncated file because we are way under the filesystem block size.

But I'll fix it up.  Can't hurt.